apps/server/src/routes/ai.ts

```
@@ -1,13 +1,67 @@
+    switch (action) {
+      case Tools.ListThreads:
+        const threads = await Promise.all(
+          (await driver.list({ folder: 'inbox', maxResults: 5 })).threads.map((thread) =>
+            driver.get(thread.id).then((thread) => ({
+              id: thread.latest?.id,
+              subject: thread.latest?.subject,
+              sender: thread.latest?.sender,
+              date: thread.latest?.receivedOn,
+            })),
+          ),
+        );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)